addons/web_tour/static/src/debug/debug_manager.js

```diff
@@ -0,0 +1,89 @@
+/** @odoo-module **/
+
+import { browser } from "@web/core/browser";
+import { useService } from "@web/core/hooks";
+import { debugRegistry } from "@web/debug/debug_registry";
+import utils from "web_tour.utils";
+
+function debugDisableTourItem(env) {
+  if (!env.services.user.isSystem) {
+    return {};
+  }
+  const activeTours = env.services.tour.getActiveTours();
+  if (activeTours.length === 0) {
+    return {};
+  }
+  return {
+    type: "item",
+    description: env._t("Disable Tours"),
+    callback: async () => {
+      await env.services.orm.call("web_tour.tour", "consume", [activeTours]);
+      for (const tourName of activeTours) {
+        browser.localStorage.removeItem(utils.get_debugging_key(tourName));
+      }
+      browser.location.reload();
+    },
+    sequence: 33,
+  };
+}
+
+function debugStartTourItem(env) {
+  if (!env.services.user.isSystem) {
+    return {};
+  }
+  return {
+    type: "item",
+    description: env._t("Start Tour"),
+    callback: async () => {
+      env.services.dialog.open(ToursDialog);
+    },
+    sequence: 32,
+  };
+}
+
+debugRegistry.add("web_tour.start_tour", debugStartTourItem);
+debugRegistry.add("web_tour.disable_tour", debugDisableTourItem);
+
+class ToursDialog extends owl.Component {
```

Comment:
this component could be moved in its own file (still in the debug folder), and its xml could be in a file with same name